src/configure-nilrt-snac

```diff
@@ -141,6 +141,29 @@ remove_niauth() {
 }
 
 
+configure_tmux() {
+	log INFO Configuring tmux to be the default shell for both SSH and local
```

Comment:
Consider: This is more verbose than above, where we just say we're configuring opkg or NTP but don't say how. Not sure what level of detail we ought to provide, but I'd expect minimal would suffice.

Reply:
If you look at the `configure_wireguard()` function, while the log lines are short, there are multiple of them explaining what is happening as it configures it. I can shorten this to a single short log, break it into multiple log lines line `configure_wireguard()`, or leave it as is, whatever is decided.